release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1492,7 +1492,7 @@ The `authentication` and `openshift-apiserver` Operators now ignore the `oauth-a
 
 * This update fixes the sort order and color of `ImageManifestVuln` objects for the Quay Security Operator. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942716[*BZ#1942716*])
 
-* Previously, the *Topology* view in the Developer perspective did not load if OpenShift namespace templates were not available because the Samples Operator was not installed. This update fixes the issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949810[*BZ#1949810*])
+* Previously, the *Topology* view showed an error or a blank page while loading because the OpenShift Samples Operator was not installed. So there were no templates to the OpenShift namespace. The current release fixes this issue. The *Topology* view now loads as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949810[BZ#1949810])
```

Comment:
```suggestion
* Previously, the *Topology* view showed an error or a blank page while loading because the OpenShift Samples Operator was not installed and there were no templates to the OpenShift namespace. The current release fixes this issue. The *Topology* view now loads as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949810[BZ#1949810])
```